### ISSUE TITLE
Is there a list of reference papers about conv-lstm in examples/asr_wsj?

### ISSUE THREAD

**Issue by sephiroce** [freewym/espresso#26]
Hello,
before I build up speech-transformer models, I first tried to train the default model using run.sh in examples/asr_wsj. The accuracy is WER 12.35% for test_92 w/o LM.
this is somewhat behind the accuracy of speech-transformer (WER 10.92%) but it is still competitive.

Could you explain which references you referred to make the training script?

**Comment by freewym** [comment]
Where did you see the 10.92% number? You can find the paper explaining Espresso [here](https://arxiv.org/pdf/1909.08723.pdf).
 Actually the number w/o LM can be better, but the current LSTM-based recipe is optimized for the results w/ LM.

**Comment by sephiroce** [comment]
I mean the 10.92% number in the original speech transformer paper. 
Table 1 in http://150.162.46.34:8080/icassp2018/ICASSP18_USB/pdfs/0005884.pdf.
And, I apologize for having asked the results already mentioned in the paper.
I checked the performance and the architecture of conv-lstm in section 4 and 2.3 respectively. Thanks a lot :)

I decoded with char-lm, I couldn't apply coverage-weight option since espresso can not recognize the argument.

This is the result.
|               |       dev93   | |       |       |    eval92     | |       |       |
|---------------|--------|--------|-------|-------|--------|--------|-------|-------|
|               | WER    | Sub    | Ins   | Del   | WER    | Sub    | Ins   | Del   |
| Espresso LSTM | 15.72% | 12.39% | 1.71% | 1.62% | 12.58% | 9.91%  | 1.49% | 1.19% |
| + CharLM      | 8.29%  | 6.62%  | 1.01% | 0.67% | 6.42%  | 4.94%  | 1.13% | 0.34% |

Thanks for the nice toolkit and training recipe!

**Comment by freewym** [comment]
I have removed the --coverage-weight option as I later found --eos-factor is enough. Will push a fix. You  can use look-ahead word LM which is better and also the default in the wsj recipe

**Comment by sephiroce** [comment]
Yes, I'm training word lm and conv-lstm with --fp16 option since my gpu memory is not sufficient to use the recipe as it is.
